searchcore/src/vespa/searchcore/proton/documentmetastore/documentmetastore.h

```diff
@@ -68,6 +68,8 @@ class DocumentMetaStore final : public DocumentMetaStoreAttribute,
 
     MetaDataStore       _metaDataStore;
     TreeType            _gidToLidMap;
+    Iterator            _gid_to_lid_map_write_itr; // Iterator used for all updates of _gidToLidMap
+    SerialNum           _gid_to_lid_map_write_itr_prepare_serial_num;
```

Comment:
Consider adding a comment on how this serial num relates to _gid_to_lid_map_write_itr.